### PR TITLE
Component/comp table row

### PR DESCRIPTION
**Pull request by harryray**
Comp table row component added.

Scss-lint.yml edited to disable PropertySortOrder as it seemed to have glitched and was failing to lint correctly.

image

### CODE REVIEW

File: vendor/assets/stylesheets/ustyle/components/_comp-table-row.scss [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+    transform-origin: center;
+    fill: #0c72bf;
+
+    .us-ct-row__key-details-toggle-open & {
+      transform: translateY(5px) rotate(-90deg);
+    }
+  }
+
+  &__key-details-info {
+    padding: 0 15px;
```
Comment (annez) [inline vendor/assets/stylesheets/ustyle/components/_comp-table-row.scss]: `15px` is `$gutter-width / 2` which is our default grid gutter

File: vendor/assets/stylesheets/ustyle/components/_comp-table-row.scss [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    border-bottom: 1px solid $c-keylinegrey;
+  }
+
+  &__title-text {
+    margin: 0 15px 0 0;
+    font-weight: bold;
+  }
+
+  &__product {
+    position: relative;
```
Comment (annez) [inline vendor/assets/stylesheets/ustyle/components/_comp-table-row.scss]: Is this needed? Why do we set position here?

File: vendor/assets/stylesheets/ustyle/components/_comp-table-row.scss (hunk 1 above)
Comment (harryray) [reply vendor/assets/stylesheets/ustyle/components/_comp-table-row.scss]: Updated!

File: vendor/assets/stylesheets/ustyle/components/_comp-table-row.scss (hunk 2 above)
Comment (harryray) [reply vendor/assets/stylesheets/ustyle/components/_comp-table-row.scss]: It's needed for some responsive functionality on tablets (absolute positioning on a child element)

**Comment by harryray** [comment]
@annez are there any outstanding changes on this PR or can it be merged? :)

**Pull request merged**